examples/gluon_alexnet/README.md

```diff
@@ -0,0 +1,219 @@
+                                            activation='relu'))
+                self.features.add(nn.Conv2D(256, kernel_size=3, padding=1,
+                                            activation='relu'))
+                self.features.add(nn.MaxPool2D(pool_size=3, strides=2))
+                self.features.add(nn.Flatten())
+                self.features.add(nn.Dense(4096, activation='relu'))
+                self.features.add(nn.Dropout(0.5))
+                self.features.add(nn.Dense(4096, activation='relu'))
+                self.features.add(nn.Dropout(0.5))
+            self.output = nn.Dense(classes)
+
+    def hybrid_forward(self, F, x):
+        x = self.features(x)
+        x = self.output(x)
+        return x
+
+class AlexnetService():
+...
+    def inference(self, data):
+        net = HybridAlexNet()
+        net.hybridize()
+        net.load_params(self.param_filename, ctx=self.ctx)
+        output = net(data)
+        return output.argmax()
+...
+```
+To understand how `.hybridize()` is useful and its internals, refer to 
+[Gluon's tutorial on Hybridize](http://gluon.mxnet.io/chapter07_distributed-learning/hybridize.html).
+
+Similar to Imperative models, this model doesn't require `Symbols` as the call to `.hybridize()` compiles the nerual-net. 
+This would store the `mxnet.symbols` implicitly. 
+
+To test this model, refer to the below section on [how to test your gluon models](#how-to-test-your-gluon-models).
+## How to test your gluon models
+To test the gluon models, we need to do the following
+1. Make sure you have the latest version of `mxnet-model-server`
+```bash
+pip install mxnet-model-server
+``` 
+2. Create a model archive file. Refer [How to create a model archive](#how-to-create-a-model-archive) 
+for that.
+3. Start the model-server with the newly create model archive.
+```bash
+mxnet-model-server --models gluon=gluon.model
+```
+4. Test this service by running requests from another terminal, as follows:
+```bash
+curl -O https://s3.amazonaws.com/model-server/inputs/kitten.jpg
+curl -X POST http://127.0.0.1:8080/gluon/predict -F "data=@kitten.jpg"
+```
+
+The expected output is
+```text
+{"prediction":[{"class":"lynx,","probability":0.9411474466323853},{"class":"leopard,","probability":0.016749195754528046},{"class":"tabby,","probability":0.012754007242619991},{"class":"Egyptian","probability":0.011728651821613312},{"class":"tiger","probability":0.008974711410701275}]}
+```
+
+## How to create a model archive
+1. Go to `mxnet-model-server/examples/gluon_alexnet` folder.
+```bash
+git clone https://github.com/awslabs/mxnet-model-server.git
+```
+
+```bash
+# Go to the example
+cd mxnet-model-server/examples/gluon_alexnet
+```
+
+```bash
+# Download the alexnet-params file. This is used when using non-pretrained gluon model.
+wget https://s3.amazonaws.com/gluon-mms-model-files/alexnet.params
+```
+
+2. Copy relavant files into a model archive
+```bash
+# Create model archive
+zip gluon.model *
```

Comment:
Users should not create model archive with the ZIP utility... they should use the model exprt CLI to do that.